lib/floe/workflow/choice_rule/data.rb

```diff
@@ -4,14 +4,14 @@ module Floe
   class Workflow
     class ChoiceRule
       class Data < Floe::Workflow::ChoiceRule
-        TYPES      = ["String", "Numeric", "Boolean", "Timestamp", "Present", "Null"].freeze
+        TYPES      = {"String" => :is_string?, "Numeric" => :is_numeric?, "Boolean" => :is_boolean?, "Timestamp" => :is_timestamp?, "Present" => :is_present?, "Null" => :is_null?}.freeze
```

Comment:
Changing `TYPES` to a hash with types and also validation methods looks odd and then you have to do TYPES.keys below.  Could you build the type check method dynamically from the type like `send("is_#{type.downcase}?")?

Reply:
`TYPES[type]` returning a frozen string seems simpler than `is_#{type.downcase}?`, but I can try that out.

Reply:
Yeah another frozen hash with validations works too